pages/02.administer/60.extend/20.helpers/helpers.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: App helpers
 template: docs
-redirect: '/packaging_apps_helpers'
+redirect: '/app_scripts_helpers'
```

Comment:
So eeeh I'm confused, the old urls had some sort of consistency like all starting with `/packaging`, but in the proposed changes, some start with `/packaging` and some start with `/app_stuff`. I don't mind changing these but 1) it would be nice if there were some sort of consistent convention and 2) the current URLs are used in various places like packaging linter, forum posts, other doc pages, github READMEs etc, so we should add some `aliases:` entries for backward compatibility

Reply:
Ah ! Alright. I don't really care about the URLs themselves, they could stay unchanged actually. I cared about the visible structure in the left panel. I could undo those changes.

Reply:
The url changes are now more consistent with the existing urls.